modules/core/src/main/scala/org/scalasteward/core/scalafmt/package.scala

```diff
@@ -23,7 +23,7 @@ import org.scalasteward.core.util.Nel
 package object scalafmt {
   val latestScalafmtVersion: Version = Version("2.0.1")
   val scalafmtGroupId = "org.scalameta"
-  val scalafmtArtifactId = "scalafmt"
+  val scalafmtArtifactId = "scalafmt-core"
```

Comment:
Note)
`"org.scalameta" %% "scalafmt"` is a dummy artifact and not exist.
To fetch the project URL for scalfmt, it should be real artifact `scalafmt-core`.

Reply:
What happens if a project actually depends on `scalafmt-core`? I think Scala Steward will only update the version number in `.scalafmt.conf` then.

Reply:
Correct.
I think Steward updates both `.scalafmt.conf` and dependencies at same time.
Fixed in https://github.com/fthomas/scala-steward/pull/820/commits/c2341c7d034dea563bc0f6f638ddd7773ad1f4f5
And example of PR is https://github.com/exoego/scala-steward-test-project/pull/34/files